src/app/zap-templates/zcl/data-model/chip/drlc-cluster.xml

```diff
@@ -139,70 +121,39 @@ limitations under the License.
     <item fieldId="6" name="HeatingSourceControl" type="HeatingSourceControlStruct" optional="true"/>
   </struct>
 
-  <!-- This does not quite match the spec, which has LoadControlEventStruct as a
-       fabric-scoped struct with some fabric-sensitive fields, but that doesn't
-       really make sense in all the places where the struct is used, and the spec
-       does not really define other things in a way that would work with that at
-       the moment. -->
   <struct name="LoadControlEventStruct">
     <cluster code="0x0096"/>
     <item fieldId="0" name="EventID" type="octet_string" length="16"/>
     <item fieldId="1" name="ProgramID" type="octet_string" length="16" isNullable="true"/>
-    <item fieldId="2" name="Control" type="EventControlBitmap" />
-    <item fieldId="3" name="DeviceClass" type="DeviceClassBitmap" />
-    <item fieldId="4" name="EnrollmentGroup" type="int8u" optional="true" />
-    <item fieldId="5" name="Criticality" type="CriticalityLevelEnum" defaut="0" />
-    <item fieldId="6" name="StartTime" type="epoch_s" isNullable="true" />
-    <item fieldId="7" name="Transitions" array="true" type="LoadControlEventTransitionStruct" />
+    <item fieldId="2" name="Status" type="LoadControlEventStatusEnum" defaut="0" />
+    <item fieldId="3" name="Control" type="EventControlBitmap" />
+    <item fieldId="4" name="Criticality" type="CriticalityLevelEnum" defaut="0" />
+    <item fieldId="5" name="StartTime" type="epoch_s" isNullable="true" />
+    <item fieldId="6" name="Transitions" array="true" type="LoadControlEventTransitionStruct" />
   </struct>
 
   <struct name="LoadControlProgramStruct">
     <cluster code="0x0096"/>
     <item fieldId="0" name="ProgramID" type="octet_string" length="16"/>
     <item fieldId="1" name="Name" type="long_char_string" length="32"/>
-    <item fieldId="2" name="EnrollmentGroup" type="int8u" isNullable="true" min="1" max="0xFF"/>
-    <item fieldId="3" name="RandomStartMinutes" type="int8u" isNullable="true" min="0x00" max="0x3C"/>
-    <item fieldId="4" name="RandomDurationMinutes" type="int8u" isNullable="true" min="0x00" max="0x3C"/>
+    <item fieldId="2" name="RandomStartMinutes" type="int8u" isNullable="true" min="0x00" max="0x3C"/>
+    <item fieldId="3" name="RandomDurationMinutes" type="int8u" isNullable="true" min="0x00" max="0x3C"/>
   </struct>
 
-  <cluster apiMaturity="provisional">
+  <cluster>
```

Comment:
This needs to stay `apiMaturity="provisional"` until we go through SVE with it, no?  Once you remove that annotation, your bar for making any XML changes goes _way_ up, and I don't get the sense this spec is that stable yet.